samcli/local/init/templates/cookiecutter-aws-sam-hello-powershell/{{cookiecutter.project_name}}/src/Function.ps1

```diff
@@ -0,0 +1,17 @@
+# PowerShell script file to be executed as a AWS Lambda function. 
+# 
+# When executing in Lambda the following variables will be predefined.
+#   $LambdaInput - A PSObject that contains the Lambda function input data.
+#   $LambdaContext - An Amazon.Lambda.Core.ILambdaContext object that contains information about the currently running Lambda environment.
+#
+# The last item in the PowerShell pipeline will be returned as the result of the Lambda function.
+#
+# To include PowerShell modules with your Lambda function, like the AWSPowerShell.NetCore module, add a "#Requires" statement 
+# indicating the module and version.
+
+#Requires -Modules @{ModuleName='AWSPowerShell.NetCore';ModuleVersion='3.3.221.0'}
+
+# This line publishes the input to CloudWatch logs so you know what triggered the function
+Write-Host (ConvertTo-Json -InputObject $LambdaInput -Compress -Depth 5)
```

Comment:
Could you change this to return a `hello world` or something like Python/Node/Ruby does: 

```json
{"message": "hello world"}
```

;)